operations/github/onboarding/onboarding.sh

```diff
@@ -69,16 +77,15 @@ function create_team() {
     # Debugging: print the JSON data being sent
     echo "Creating team $name with data: $data"
 
-    # Perform the API request
+     # Perform the API request
     http_code=$(github_api -X POST -d "$data" "https://api.github.com/orgs/$org/teams")
-    
-    # Debugging: print the HTTP response code and response body
-    echo "HTTP response code: $http_code"
-    echo response.json
 
-    if [ "$http_code" -ne 201 ]; then
+    echo "HTTP response Code: $http_code"
+    
+    if [ "$http_code" -ne $HTTP_RESPONSE_CODE_CREATED ]; then
```

Comment:
Normally variable references are surrounded by "...". It's probably fine in this specific case, but I *strongly* recommend that you run "shellcheck" on all shell scripts & fix what it finds.